btcec/schnorr/musig2/nonces.go

```diff
@@ -392,18 +373,71 @@ func AggregateNonces(pubNonces [][PubNonceSize]byte) ([PubNonceSize]byte, error)
 	if err != nil {
 		return finalNonce, err
 	}
+
 	combinedNonce2, err := combineNonces(func(n [PubNonceSize]byte) []byte {
 		return n[btcec.PubKeyBytesLenCompressed:]
 	})
 	if err != nil {
 		return finalNonce, err
 	}
 
-	copy(finalNonce[:], combinedNonce1.SerializeCompressed())
+	copy(finalNonce[:], JacobianToByteSlice(combinedNonce1))
 	copy(
 		finalNonce[btcec.PubKeyBytesLenCompressed:],
-		combinedNonce2.SerializeCompressed(),
+		JacobianToByteSlice(combinedNonce2),
 	)
 
 	return finalNonce, nil
 }
+
+// NonceToJacobian parses the nonce as a secp.Pubkey and converts the pubkey
+// to the passed result. If the nonce is a zero slice, the infinityPoint is
+// returned.
+func NonceToJacobian(nonce []byte, result *btcec.JacobianPoint) error {
```

Comment:
How about we make this a bit more generic, and move it up to the `btcec` module level. 

So we'd end up w/ something like:
```go
func ParseJacobian(point []byte) (*btcec.JacobianPoint, error) {
    point, err := ParsePubKey(point)
    if err != nil {
        return err
    }
    
   var j btcec.JacobianPoint
    point.AsJacobian(j)
 
    return &j, nil
}
```